onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -489,7 +492,7 @@ Status QNNExecutionProvider::Compile(const std::vector<FusedNodeAndGraph>& fused
       // dump fused_node.OpType() as metadata in context cache binary file, so that we can validate it in GetCapability
       ORT_RETURN_IF_ERROR(qnn_backend_manager_->DumpQnnContext(context_cache_pathstring,
                                                                GetFileNameFromModelPath(graph_viewer.ModelPath()),
-                                                               fused_node.OpType()));
+                                                               graph_viewer.Name()));
```

Comment:
The comment above this statement (line 492) should be updated since we're not passing in `fused_node.OpType()` anymore.